adopters.md

```diff
@@ -0,0 +1,13 @@
+# Adopters
+
+This document lists the environments you can use OSM in as well as some additional OSM usage guidelines.
+
+## Environments
```

Comment:
Adopters and environments seem orthogonal to me.

It would make more sense to either:
1. rename this file to environments.md
2. Update the wording to document adoption of this project (e.g. https://github.com/projectcontour/contour/blob/main/ADOPTERS.md)

Reply:
@shashankram updated PR to use environments.md since we don't have the content available to create adopters.md